app/Controllers/ReservationsAdminController.php

```
@@ -121,12 +121,14 @@ public function update(Request $request, Response $response, mysqli $db, int $id
         if ($dataFineRichiesta !== '' && !preg_match('/^\d{4}-\d{2}-\d{2}$/', $dataFineRichiesta))
             $dataFineRichiesta = '';
 
-        $today = date('Y-m-d');
+        $today = \App\Support\DateHelper::today();
         if ($start === '') {
             $start = $today;
         }
         if ($end === '') {
-            $end = date('Y-m-d', strtotime($start . ' +1 month'));
+            $loanDays = (int) ((new \App\Models\SettingsRepository($db))->get('loans', 'loan_duration_days', '30') ?? 30);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '129', column '37'). (undefined)

(MissingImport)